docs/source/apis/api_leveling.md

```diff
@@ -0,0 +1,57 @@
+# Llama Stack API Stability Leveling
+
+In order to provide a stable experience in Llama Stack, the various APIs need different stability levels indicating the level of support, backwards compatability, and overall production readiness.
+
+## Different Levels
+
+### v1alpha
+
+- Little to no expectation of support between versions
+- Breaking changes are permitted
+- Datatypes and parameters can break
+- Routes can be added and removed
+
+### v1beta
+
+- API routes ensured between versions
+- Parameters and return types are not
+- API, besides minor fixes and adjustments, should be _almost_ v1. Changes should not be drastic.
+
+### v1 (stable)
+
+- Considered stable
```

Comment:
I think this is key: what are the criteria to graduate APIs from alpha to beta to stable?

Reply:
Thanks for adding those!